plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/controller/chat/messenger/ChatPromptHandler.kt

```diff
@@ -143,6 +144,24 @@ class ChatPromptHandler(private val telemetryHelper: TelemetryHelper) {
             }
     }
 
+    private fun extractCodeBlockLanguage(message: String): String {
+        // This fulfills both the cases of unit test generation(java, python) and general use case(Non java and Non python) languages.
+        val codeBlockStart = message.indexOf("```")
+        if (codeBlockStart == -1) {
+            return "plaintext"
```

Comment:
Can we assign plaintext to a variable?

Reply:
Yes I added a new variable `defaultTestGenResponseLanguage = plaintext` and used instead of plaintext.